.golangci.yaml

```diff
@@ -0,0 +1,38 @@
+run:
+  concurrency: 4
+  timeout: 5m
+  tests: true
+  skip-dirs:
+    - vendor
+    - test
+  modules-download-mode: vendor
+linters:
+  disable-all: true
+  enable:
+    - errcheck
+    - bodyclose
+    - exportloopref
+    - gosimple
+    - govet
+    - ineffassign
+    - misspell
+    - nolintlint
+    - nosprintfhostport
+    - staticcheck
+    - tenv
+    - typecheck
+    - unconvert
+    - unused
+    - wastedassign
+    - whitespace
+linters-settings:
+  misspell:
+    locale: US
+    ignore-words:
+      - NRO
+      - nro
+      - numaresource
+      - numa
+      - NUMA
+      - behaviour
+      - flavour
```

Comment:
this is questionable, we seem to prefer the american spelling (vs the english one) but it can be changed later

Reply:
deleted and changed the typos.